modules/social_features/social_comment/config/install/views.view.comment.yml

```diff
@@ -0,0 +1,1974 @@
+langcode: en
```

Comment:
@dmytriikaiun  Shouldn't we add config `views.view.comment.yml` to the **optional** folder as it did with original config `/core/modules/comment/config/optional/views.view.comment.yml`?

Reply:
@Ressinel, the configuration in the `optional` directory cannot override the existing configuration. As we already have views config in the `comment` module, the `social_comment` will not install a new one. That is why we should use `install`. For the same reason, I guess, the [view](https://github.com/goalgorilla/open_social/blob/main/modules/social_features/social_user/config/install/views.view.user_admin_people.yml) of users is also in the install directory.

Reply:
It makes sense, I agree.